src/questions/makerdao.yaml

```
@@ -0,0 +1,245 @@
+- uuid: cb274dad-bb8d-484a-83a5-486530ddff69
+  type: SingleChoice
+  content: What are the two tokens in MakerDAO's two token system?
+  hint: NoHint
+  explanation: MKR and DAI
+  answerKeys:
+    - A
+  difficultyLevel: "low"
+  choices:
+    - content: MKR and DAI
+      key: A
+      
+    - content: DAI and SAI
+      key: B
+      
+    - content: DAI and VAI
+      key: C
+      
+    - content: MKR and DAO
+      key: D
+      
+- uuid: 567a6a91-b11f-4823-908f-a3a7903ce614
+  type: SingleChoice
+  content: How is DAI borrowed?
+  hint: NoHint
+  explanation: DAI is borrowed by opening an overcollateralized debt position by creating a maker vault
+  answerKeys:
+    - D
+  difficultyLevel: "Medium"
+  choices:
+    - content: DAI can no longer be borrowed in MakerDAO
+      key: A
+      
+    - content: By buying it at a 1:1 ratio with USD
+      key: B
+      
+    - content: By requesting MakerDAO officials
+      key: C
+      
+    - content: By opening an overcollateralized borrow position in a Maker vault
+      key: D
+      
+- uuid: 27b70dc5-b43e-46f9-a311-2d545ad3794c
+  type: MultipleChoice
+  content: All Maker vaults have the same set of risk parameters (for example, dust)
+  hint: NoHint
+  explanation: Every Maker Vault has its own risk parameters.
+  answerKeys:
+    - B
+  difficultyLevel: "Medium"
+  choices:
+    - content: "True"
+      key: A
+      
+    - content: "False"
+      key: B
+      
+- uuid: 89b629ad-2c5f-4aee-bb15-7218360ba572
+  type: SingleChoice
+  content: What is dust in MakerDAO?
+  hint: NoHint
+  explanation: The minimum vault debt (or) the minimum possible debt of a vault is called dust. 
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: Minimum amount of DAI that must be borrowed from a given vault
+      key: A
+      
+    - content: The fees paid by user to stabilize the protocol
+      key: B
+      
+    - content: The interest paid by user on their debt
+      key: C
+      
+    - content: The interest received by user for depositing in Maker vaults
+      key: D
+          
+- uuid: 8af1460a-a78f-4a08-8ad3-ba5e1ec27d39
+  type: MultipleChoice
+  content: What are the use cases of DSR?
+  hint: NoHint
+  explanation: Users can deposit DAI into DSR (Dai Savings Rate) to earn interest, the Maker protocol can use the DSR to maintain the peg of DAI by increasing and decreasing the rate of interest.
+  answerKeys:
+    - A
+    - D
+  difficultyLevel: "Medium"
+  choices:
+    - content: Users can earn interest on deposited DAI
+      key: A
+      
+    - content: DSR is a function used to liquidate bad borrows
+      key: B
+      
+    - content: Users can borrow more DAI using DSR
+      key: C
+      
+    - content: The protocol can use DSR to maintain a stable peg for DAI
```

```suggestion
    - content: The protocol can use DSR to maintain a stable peg for DAI
```

Expand on it by explaining that DSR impacts the supply/demand and can be used to balance supply and demand which then stabilizes peg for DAI